docs/css/README.md

```diff
@@ -1,14 +1,201 @@
-- [Using Sass](sass.md)
+This boilerplate uses [`styled-components`](https://github.com/styled-components/styled-components) :nail_care:
+for styling react components. `styled-components` allows you to write actual CSS inside your JavaScript,
+enabling you to use the [full power of CSS](https://github.com/styled-components/styled-components/blob/master/docs/css-we-support.md) :muscle:
+without mapping between styles and components.
+There are many ways to style react applications, but many find `styled-components`
+to be a more natural approach to styling components.
+Watch this video for a comparison and to see how it enforces best practices!
+
+[![Styled-components: Enforcing best practices](http://img.youtube.com/vi/jaqDA7Btm3c/0.jpg)](https://youtu.be/jaqDA7Btm3c)
+
+To supplement `styled-components`, this boilerplate also uses
+[`sanitize.css`](https://github.com/jonathantneal/sanitize.css)
+to make browsers render all elements more consistently and in line with modern standards,
+it's a modern alternative to CSS resets. More info available on the [`sanitize.css` page](sanitize.md).
+
+## CSS Support
+
+We support and recommend the use of [`styled-components`](#components).
+We also support the use of CSS [stylesheets](#stylesheet).
+
+There are many ways to style web applications, unfortunately, we cannot support them all.
+However, you can integrate the following styling methods:
+- [CSS Modules](#css-modules)
+- [Sass](#sass)
+
+## `styled-components`
+
+Below creates two styled react components (`<Title>`, `<Wrapper>`) and renders them
+as children of the `<Header>` component:
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+
+// Create a <Title> react component that renders an <h1> which is
+// centered, palevioletred and sized at 1.5em
+const Title = styled.h1`
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+`;
+
+// Create a <Wrapper> react component that renders a <section> with
+// some padding and a papayawhip background
+const Wrapper = styled.section`
+  padding: 4em;
+  background: papayawhip;
+`;
+
+// Use them like any other React component – except they're styled!
+function Header(props) {
+  return (
+    <Wrapper>
+      <Title>Hello {props.name}, this is your first styled component!</Title>
+      ...
+    </Wrapper>
+  );
+}
+
+```
+
+*(The CSS rules are automatically vendor prefixed, so you don't have to think about it!)*
+
+For further examples see the
+[official documentation](https://github.com/styled-components/styled-components)
+
+## Stylesheet
+
+[Webpack](https://webpack.js.org/) allows you to import more than JavaScript.
+Using the [`css-loader`](https://webpack.js.org/loaders/css-loader/) you can import CSS
+into a JavaScript:
+
+**`Button.css`**
+```css
+.danger {
+  background-color: red;
+}
+```
+
+**`Button.js`**
+```js
+import React from 'react';
+import './Button.css'; // Tell Webpack that Button.js uses these styles
+
+class Button extends Component {
```

Comment:
Most definitely, good pickup. I will go with `React.Component` as it is used heavily in the react docs.